github_pr.py

```diff
@@ -369,8 +398,23 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+    !!!This check only looks at comments AFTER the latest commit, to validate that the
+    code has been most recently peer reviewed.!!!
+
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-non-owner-merger
+        This will check to make sure that the owner can not apply a shipable comment on their own code
+    github-pr check-condition -r dataxu/dcommand -n84 --condition-approved-mergers-file=<MAINTAINERS FILE>
+        This takes the path to the MAINTAINERS file inside the repo
```

Comment:
do we want to mention that the MAINTAINERS file checked should be from the master branch, regardless of which branch the PR is coming from? or do we handle that implicitly in the jenkins job when we call this script?